core/src/main/java/bisq/core/trade/protocol/BuyerAsTakerProtocol.java

```diff
@@ -65,6 +65,7 @@ public BuyerAsTakerProtocol(BuyerAsTakerTrade trade) {
 
         Offer offer = checkNotNull(trade.getOffer());
         processModel.getTradingPeer().setPubKeyRing(offer.getPubKeyRing());
+        processModel.getTradeManager().requestPersistence();
```

Comment:
I'm getting a NPE here, processModel.tradeManager is null

Reply:
Ah in the constructor it must not be called as trademanager is not set yet. I will remove that.